README.md

```diff
@@ -1 +1,97 @@
+ID, access key, and resource group name.
+
+1. Create a configuration file (for example, `maxcompute.properties`) in Presto's `etc/catalog` directory:
+
+```properties
+# MaxCompute Connector
+connector.name=maxcompute
+# MaxCompute project name, connections support only this project
+odps.project.name=<Your MaxCompute Project Name>
+# Alibaba Cloud authentication parameter
+odps.access.id=<Your Alibaba Cloud Access ID>
+# Alibaba Cloud authentication parameter
+odps.access.key=<Your Alibaba Cloud Access Key>
+# MaxCompute Endpoint
+odps.end.point=<Your MaxCompute Endpoint>
+# MaxCompute exclusive resource group name
+odps.quota.name=<Your MaxCompute Exclusive Resource Group Name>
+```
+
+2. Replace `<Your MaxCompute Project Name>`, `<Your Alibaba Cloud Access ID>`, `<Your Alibaba Cloud Access Key>`, `<Your MaxCompute Service Endpoint>`,
+and `<Your MaxCompute Exclusive Resource Group Name>` with the values for your MaxCompute environment.
+
+### Accessing Multiple MaxCompute Projects
+
+To access multiple MaxCompute projects simultaneously, create separate configuration files for each project. For example, if you have two projects, one for "sales" (`sales`)
+and another for "analysis" (`analytics`), then you can create `sales.properties` and `analytics.properties` configuration files. Specify a different `odps.project.name` in each
+file:
+
+- `sales.properties` for the sales project.
+- `analytics.properties` for the analysis project.
+  With this setup, `sales` and `analytics` catalogs will be created in Presto respectively.
+
+### Data Type Mapping
+
+When using the MaxCompute connector, MaxCompute's data types are mapped to Presto's data types. The table below shows this mapping relationship and its corresponding Presto
+types:
+
+| MaxCompute Data Type | Presto Data Type |
+|----------------------|------------------|
+| BOOLEAN              | BOOLEAN          |
+| TINYINT              | TINYINT          |
+| SMALLINT             | SMALLINT         |
+| INT                  | INTEGER          |
+| BIGINT               | BIGINT           |
+| FLOAT                | REAL             |
+| DOUBLE               | DOUBLE           |
+| DECIMAL              | DECIMAL          |
+| STRING               | VARCHAR          |
+| VARCHAR              | VARCHAR          |
+| JSON                 | VARCHAR          |
+| CHAR                 | CHAR             |
+| BINARY               | VARBINARY        |
+| DATE                 | DATE             |
+| DATETIME             | TIMESTAMP        |
+| TIMESTAMP            | TIMESTAMP        |
+| TIMESTAMP_NTZ        | TIMESTAMP        |
+| DECIMAL              | DECIMAL          |
+| ARRAY                | ARRAY            |
+
+Please note that some MaxCompute data types (such as `MAP`, `STRUCT`) are not yet fully supported in the current version of the connector. We plan to continue refining support for
```

Comment:
```suggestion
Note: Some MaxCompute data types (such as `MAP`, `STRUCT`) are not fully supported in the current version of the connector.
```
In general, documentation should describe the current state and does not use future tense. Suggest deleting the second sentence.